examples/cpp/route_guide/Makefile

```diff
@@ -86,7 +86,7 @@ ifneq ($(HAS_VALID_PROTOC),true)
 	@echo "Please install Google protocol buffers 3.0.0 and its compiler."
 	@echo "You can find it here:"
 	@echo
-	@echo "   https://github.com/google/protobuf/releases/tag/v3.0.0-alpha-1"
+	@echo "   https://github.com/google/protobuf/releases/tag/v3.0.0-beta-2"
```

Comment:
They are related. You are modifying the files examples/cpp/helloworld/Makefile and examples/cpp/routeguide/Makefile, and which also requires updating the copyright, otherwise our test suite checking the copyright will fail.
If you can't update the copyright, I won't be able to merge this PR - can't afford to break the tests (and I will probably need to go a more complicated way to get these changes in).